pkg/networkmanager/networking.scss

```diff
@@ -1,6 +1,8 @@
 @use "ct-card";
 @use "page";
 @import "global-variables";
+// Needed for Firewall's add services dialog's filter layout
```

Comment:
Note that I've added a comment as to why we're importing this CSS here, to prevent this mistaken removal in the future.